rfhub2/cli/keywords/keywords_importer.py

```diff
@@ -32,26 +32,17 @@ def delete_all_collections(self) -> Set[int]:
         """
```

Comment:
This whole method is not needed, due to having 
```
self.client.delete_all_collections()
```

Reply:
I have removed it, it should work well still. The only thing is the coverage. Have no idea how to cover line: 110 in collections.py

Reply:
Probably it's safe to go without else. About coverage, don't try to cover everything for sake of it :)